packages/suite-desktop-core/e2e/tests/settings/t2b1-device-settings.test.ts

```diff
@@ -10,8 +10,9 @@ test.describe.serial('T2B1 - Device settings', { tag: ['@group=settings'] }, ()
         emulatorStartConf: { version: '2-latest', model: 'T2B1', wipe: true },
     });
 
-    test.beforeEach(async ({ onboardingPage, settingsPage }) => {
+    test.beforeEach(async ({ onboardingPage, dashboardPage, settingsPage }) => {
         await onboardingPage.completeOnboarding();
+        await dashboardPage.discoveryShouldFinish();
```

Comment:
I had a random flaky fail on the run. I realized that the test is trying to manipulate with a setting that is dependent on the device discovery